modules/thirdBase.py

```diff
@@ -0,0 +1,55 @@
+import random
+import modules
+from modules.whosOnFirst import WhosOnFirst
+
+class ThirdBase(WhosOnFirst):
+    identifiers = ['thirdBase']
+    display_name = "Third Base"
+    manual_name = "Third Base"
+    help_text = "`{cmd} push 8i99` or `{cmd} press 66i8` to push a button."
+    module_score = 6
+    transform='rotate(180 174 174)'
+
+    def button_replace(self,v):
```

Comment:
This function is never called because you didn't change the name in both places.

Reply:
this is for an outdated commit

Reply:
See other comments.